src/main/java/software/amazon/qldb/QldbDriverImpl.java

```diff
@@ -152,43 +152,45 @@ public void execute(ExecutorNoReturn executor, RetryPolicy retryPolicy) {
                 return returnedValue;
             } catch (ExecuteException ee) {
                 // If initial session is invalid, always retry once with a new session.
-                if (ee.isRetriable && ee.isISE && retryAttempt == 0) {
+                if (ee.isRetryable() && ee.isISE() && retryAttempt == 0) {
```

Comment:
Should be `ee.isIse()`... but I prefer the full name...

Reply:
Updated to `ee.isInvalidSessionException()`